### PR TITLE
Add setting to enable multiline comment customizations

### PR DESCRIPTION
**Pull request by bobbrow**
#1539, #1100

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.15.1: March 2018
+* Support additional multiline comment patterns. #[1539](https://github.com/Microsoft/vscode-cpptools/issues/1539)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: need to add reference to issue 1100, and spell out the setting name

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.15.1: March 2018
+* Support additional multiline comment patterns. #[1100](https://github.com/Microsoft/vscode-cpptools/issues/1100),#[1539](https://github.com/Microsoft/vscode-cpptools/issues/1539)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Is the `#` supposed to be inside the `[]`?

File: Extension/package.json [hunk 3]
```diff
@@ -194,6 +194,35 @@
...
+              {
+                "type": "string",
+                "description": "The pattern that begins a multiline or single line comment block. The continuation pattern defaults to ' * ' for multiline comment blocks or this string for single line comment blocks."
+              },
+              {
+                "type": "object",
+                "properties": {
+                  "begin": {
+                    "type": "string",
+                    "description": "The pattern that begins a multiline or single line comment block"
```
Comment (sean-mcmanus) [inline Extension/package.json]: Is this supposed to end in a period?

File: Extension/package.json [hunk 4]
```diff
@@ -194,6 +194,35 @@
...
+                  },
+                  "continue": {
+                    "type": "string",
+                    "description": "The text that will be inserted on the next line when Enter is pressed inside a multiline comment block"
+                  }
+                }
+              }
+            ]
+          },
+          "description": "Defines the editor behavior for when the Enter key is pressed inside of a multiline comment block.",
```
Comment (sean-mcmanus) [inline Extension/package.json]: Should "inside of" be "inside"? It's inconsistent with the previous description.

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/CHANGELOG.md (hunk 2 above)
Comment (bobbrow) [reply Extension/CHANGELOG.md]: yep. thanks for spotting this.

File: Extension/package.json (hunk 3 above)
Comment (bobbrow) [reply Extension/package.json]: yep

File: Extension/package.json (hunk 4 above)
Comment (bobbrow) [reply Extension/package.json]: ok

**Pull request merged**